src/opera/commands/deploy.py

```diff
@@ -35,6 +39,12 @@ def deploy(args):
     storage = Storage(Path(args.instance_path).joinpath(".opera")) if args.instance_path else Storage(Path(".opera"))
     storage.write(args.csar.name, "root_file")
 
+    try:
+        num_workers = int(args.workers) if args.workers else 1
+    except ValueError:
+        print("{0} is not a valid number!".format(args.workers))
+        return 1
+
```

Comment:
This can go away by setting that parser argument type and default value.

Reply:
@matejart Can parser argument also check that the number is greater than 0? I forgot to add that part....